resources/brave-youtube-navigation-fix.js

```diff
@@ -0,0 +1,12 @@
+let previous = 0;
```

Comment:
its also strange to have global state here, but _not_ have it in the closure.  The point of the self invoking function is to hide your global state